docs/backup_and_restore.rst

```diff
@@ -521,20 +527,28 @@ source accounts, and journalist accounts. To do so, follow the steps below:
    settings, test connectivity, and set up SSH keys to allow for
    *Admin Workstation* access.
 
+   .. note::
+
+      You will need to wait approximately 10-15 minutes after installing 
```

Comment:
We could say, "You _may_ need to wait ..." instead of adding a mandatory extra 20 minutes into the process.